[2023-08-18T04:56:50.841+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dags_external_task_sensor.external_task_sensor_c manual__2023-08-18T04:55:47.721833+00:00 [queued]>
[2023-08-18T04:56:50.855+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dags_external_task_sensor.external_task_sensor_c manual__2023-08-18T04:55:47.721833+00:00 [queued]>
[2023-08-18T04:56:50.855+0000] {taskinstance.py:1308} INFO - Starting attempt 2 of 2
[2023-08-18T04:56:50.885+0000] {taskinstance.py:1327} INFO - Executing <Task(ExternalTaskSensor): external_task_sensor_c> on 2023-08-18 04:55:47.721833+00:00
[2023-08-18T04:56:50.896+0000] {standard_task_runner.py:57} INFO - Started process 2617 to run task
[2023-08-18T04:56:50.900+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dags_external_task_sensor', 'external_task_sensor_c', 'manual__2023-08-18T04:55:47.721833+00:00', '--job-id', '38', '--raw', '--subdir', 'DAGS_FOLDER/dags_external_task_sensor.py', '--cfg-path', '/tmp/tmpkvgiqa3g']
[2023-08-18T04:56:50.902+0000] {standard_task_runner.py:85} INFO - Job 38: Subtask external_task_sensor_c
[2023-08-18T04:56:50.967+0000] {task_command.py:410} INFO - Running <TaskInstance: dags_external_task_sensor.external_task_sensor_c manual__2023-08-18T04:55:47.721833+00:00 [running]> on host 05c492c53b17
[2023-08-18T04:56:51.078+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dags_external_task_sensor' AIRFLOW_CTX_TASK_ID='external_task_sensor_c' AIRFLOW_CTX_EXECUTION_DATE='2023-08-18T04:55:47.721833+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-08-18T04:55:47.721833+00:00'
[2023-08-18T04:56:51.079+0000] {external_task.py:236} INFO - Poking for tasks ['task_c'] in dag dags_branch_python_operator on 2023-08-17T22:55:47.721833+00:00 ... 
[2023-08-18T04:57:01.096+0000] {external_task.py:236} INFO - Poking for tasks ['task_c'] in dag dags_branch_python_operator on 2023-08-17T22:55:47.721833+00:00 ... 
[2023-08-18T04:57:11.116+0000] {external_task.py:236} INFO - Poking for tasks ['task_c'] in dag dags_branch_python_operator on 2023-08-17T22:55:47.721833+00:00 ... 
[2023-08-18T04:57:21.136+0000] {external_task.py:236} INFO - Poking for tasks ['task_c'] in dag dags_branch_python_operator on 2023-08-17T22:55:47.721833+00:00 ... 
[2023-08-18T04:57:31.156+0000] {external_task.py:236} INFO - Poking for tasks ['task_c'] in dag dags_branch_python_operator on 2023-08-17T22:55:47.721833+00:00 ... 
[2023-08-18T04:57:41.180+0000] {external_task.py:236} INFO - Poking for tasks ['task_c'] in dag dags_branch_python_operator on 2023-08-17T22:55:47.721833+00:00 ... 
[2023-08-18T04:57:51.196+0000] {external_task.py:236} INFO - Poking for tasks ['task_c'] in dag dags_branch_python_operator on 2023-08-17T22:55:47.721833+00:00 ... 
[2023-08-18T04:58:01.209+0000] {external_task.py:236} INFO - Poking for tasks ['task_c'] in dag dags_branch_python_operator on 2023-08-17T22:55:47.721833+00:00 ... 
[2023-08-18T04:58:11.234+0000] {external_task.py:236} INFO - Poking for tasks ['task_c'] in dag dags_branch_python_operator on 2023-08-17T22:55:47.721833+00:00 ... 
[2023-08-18T04:58:21.256+0000] {external_task.py:236} INFO - Poking for tasks ['task_c'] in dag dags_branch_python_operator on 2023-08-17T22:55:47.721833+00:00 ... 
[2023-08-18T04:58:31.286+0000] {external_task.py:236} INFO - Poking for tasks ['task_c'] in dag dags_branch_python_operator on 2023-08-17T22:55:47.721833+00:00 ... 
[2023-08-18T04:58:41.307+0000] {external_task.py:236} INFO - Poking for tasks ['task_c'] in dag dags_branch_python_operator on 2023-08-17T22:55:47.721833+00:00 ... 
[2023-08-18T04:58:51.327+0000] {external_task.py:236} INFO - Poking for tasks ['task_c'] in dag dags_branch_python_operator on 2023-08-17T22:55:47.721833+00:00 ... 
[2023-08-18T04:58:57.126+0000] {local_task_job_runner.py:292} WARNING - State of this instance has been externally set to queued. Terminating instance.
[2023-08-18T04:58:57.130+0000] {process_utils.py:135} INFO - Sending Signals.SIGTERM to group 2617. PIDs of all processes in the group: [2617]
[2023-08-18T04:58:57.131+0000] {process_utils.py:86} INFO - Sending the signal Signals.SIGTERM to group 2617
[2023-08-18T04:58:57.132+0000] {taskinstance.py:1517} ERROR - Received SIGTERM. Terminating subprocesses.
[2023-08-18T04:58:57.153+0000] {taskinstance.py:1824} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/sensors/base.py", line 253, in execute
    time.sleep(self._get_next_poke_interval(started_at, run_duration, try_number))
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1519, in signal_handler
    raise AirflowException("Task received SIGTERM signal")
airflow.exceptions.AirflowException: Task received SIGTERM signal
[2023-08-18T04:58:57.165+0000] {taskinstance.py:1350} INFO - Marking task as FAILED. dag_id=dags_external_task_sensor, task_id=external_task_sensor_c, execution_date=20230818T045547, start_date=20230818T045650, end_date=20230818T045857
[2023-08-18T04:58:57.220+0000] {standard_task_runner.py:109} ERROR - Failed to execute job 38 for task external_task_sensor_c (Task received SIGTERM signal; 2617)
[2023-08-18T04:58:57.265+0000] {process_utils.py:79} INFO - Process psutil.Process(pid=2617, status='terminated', exitcode=1, started='04:56:50') (2617) terminated with exit code 1
